### PR TITLE
Enable more distros

### PR DESCRIPTION
**Pull request by livarcocc**
@dotnet/dotnet-cli

### CODE REVIEW

**Comment by livarcocc** [comment]
@dotnet-bot test ci please

**Comment by livarcocc** [comment]
@dotnet-bot Test RHEL7.2 x64 Release Build

**Comment by livarcocc** [comment]
@dotnet-bot Test Windows_NT x86 Debug Build

**Comment by livarcocc** [comment]
@dotnet-bot Test Windows_NT x64 Release Build

**Comment by livarcocc** [comment]
@dotnet-bot Test Windows_NT x86 Debug Build

File: run-build.sh [hunk 1]
```diff
@@ -165,8 +165,8 @@ if [ $? != 0 ]; then
 fi
 
 # now execute the script
-echo "installing CLI: $dotnetInstallPath --channel \"master\" --install-dir $DOTNET_INSTALL_DIR --architecture \"$ARCHITECTURE\" $LINUX_PORTABLE_INSTALL_ARGS"
-$dotnetInstallPath --channel "master" --install-dir $DOTNET_INSTALL_DIR --architecture "$ARCHITECTURE" $LINUX_PORTABLE_INSTALL_ARGS
+echo "installing CLI: $dotnetInstallPath --channel \"master\" --version \"2.0.0-preview1-005165\" --install-dir $DOTNET_INSTALL_DIR --architecture \"$ARCHITECTURE\" $LINUX_PORTABLE_INSTALL_ARGS"
```
Comment (eerhardt) [inline run-build.sh]: (nit) this version is inconsistent with below

**Comment by livarcocc** [comment]
@dotnet-bot Test Windows_NT x64 Release Build

**Comment by eerhardt** [comment]
@livarcocc - can this be merged?  It is green.

I'd like to get this change in so I can continue getting tests enabled for `linux-x64`.

File: test/dotnet-new.Tests/GivenThatIWantANewAppWithSpecifiedType.cs [hunk 2]
```diff
@@ -23,11 +23,12 @@ public class GivenThatIWantANewAppWithSpecifiedType : TestBase
         [InlineData("C#", "web", true)]
         [InlineData("C#", "mvc", true)]
         [InlineData("C#", "webapi", true)]
-        [InlineData("F#", "console", false)]
-        [InlineData("F#", "classlib", false)]
-        [InlineData("F#", "mstest", false)]
-        [InlineData("F#", "xunit", false)]
-        [InlineData("F#", "mvc", true)]
+        // Uncomment the test below once https://github.com/dotnet/netcorecli-fsc/issues/92 is fixed.
+        //[InlineData("F#", "console", false)]
```
Comment (eerhardt) [inline test/dotnet-new.Tests/GivenThatIWantANewAppWithSpecifiedType.cs]: We shouldn't completely turn these tests off.

Instead, we should just disable them on the RIDs where it isn't supported.

See [https://github.com/dotnet/cli/blob/rel/1.0.1/test/dotnet-new.Tests/GivenThatIWantANewAppWithSpecifiedType.cs#L33-L38](https://github.com/dotnet/cli/blob/rel/1.0.1/test/dotnet-new.Tests/GivenThatIWantANewAppWithSpecifiedType.cs#L33-L38)

**Pull request merged**